src/backend/dataExports/cityLevelWeeklyGeneralResults.ts

```diff
@@ -0,0 +1,117 @@
+import { App, s3PutJsonHelper } from '../app';
+import { accumulateResultsByCity, filterResultsByCity } from './cityLevelGeneralResults';
+import { PostalCodeCityMappings, PopulationPerCity } from '../../common/model';
+import { PostalCodeLevelGeneralResultsQuery } from './postalCodeLevelGeneralResults';
+
+export async function exportCityLevelWeeklyGeneralResults(app: App) {
+  const cityLevelWeeklyGeneralResults = await fetchCityLevelWeeklyGeneralResults(app);
+  await pushCityLevelWeeklyGeneralResults(app, cityLevelWeeklyGeneralResults);
+}
+
+export async function fetchCityLevelWeeklyGeneralResults(app: App) {
+  const postalCodeLevelResultsResult = await queryPostalCodeLevelGeneralResults(app);
+
+  const postalCodeCityMappings = await app.s3Sources.fetchPostalCodeCityMappings();
+  const populationPerCity = await app.s3Sources.fetchPopulationPerCity();
+
+  const cityLevelWeeklyGeneralResults = mapCityLevelWeeklyGeneralResults(
+    postalCodeLevelResultsResult.Items,
+    postalCodeCityMappings,
+    populationPerCity,
+  );
+  return cityLevelWeeklyGeneralResults;
+}
+
+//
+// Query
+
+export const postalCodeLevelWeeklyGeneralResultsQuery = `SELECT postal_code,
+  COUNT(*) AS responses,
+  COUNT(IF ( fever = 'no', 1, NULL)) AS fever_no,
+  COUNT(IF ( fever = 'slight', 1, NULL)) AS fever_slight,
+  COUNT(IF ( fever = 'high', 1, NULL)) AS fever_high,
+  COUNT(IF ( cough = 'no', 1, NULL)) AS cough_no,
+  COUNT(IF ( cough = 'mild', 1, NULL)) AS cough_mild,
+  COUNT(IF ( cough = 'intense', 1, NULL)) AS cough_intense,
+  COUNT(IF ( general_wellbeing = 'fine', 1, NULL)) AS general_wellbeing_fine,
+  COUNT(IF ( general_wellbeing = 'impaired', 1, NULL)) AS general_wellbeing_impaired,
+  COUNT(IF ( general_wellbeing = 'bad', 1, NULL)) AS general_wellbeing_bad,
+  COUNT(IF ( breathing_difficulties = 'no', 1, NULL)) AS breathing_difficulties_no,
+  COUNT(IF ( breathing_difficulties = 'yes', 1, NULL)) AS breathing_difficulties_yes,
+  COUNT(IF ( muscle_pain = 'no', 1, NULL)) AS muscle_pain_no,
+  COUNT(IF ( muscle_pain = 'yes', 1, NULL)) AS muscle_pain_yes,
+  COUNT(IF ( headache = 'no', 1, NULL)) AS headache_no,
+  COUNT(IF ( headache = 'yes', 1, NULL)) AS headache_yes,
+  COUNT(IF ( sore_throat = 'no', 1, NULL)) AS sore_throat_no,
+  COUNT(IF ( sore_throat = 'yes', 1, NULL)) AS sore_throat_yes,
+  COUNT(IF ( rhinitis = 'no', 1, NULL)) AS rhinitis_no,
+  COUNT(IF ( rhinitis = 'yes', 1, NULL)) AS rhinitis_yes,
+  COUNT(IF ( stomach_issues = 'no', 1, NULL)) AS stomach_issues_no,
+  COUNT(IF ( stomach_issues = 'yes', 1, NULL)) AS stomach_issues_yes,
+  COUNT(IF ( sensory_issues = 'no', 1, NULL)) AS sensory_issues_no,
+  COUNT(IF ( sensory_issues = 'yes', 1, NULL)) AS sensory_issues_yes,
+  COUNT(IF ( longterm_medication	 = 'no', 1, NULL)) AS longterm_medication_no,
```

Comment:
Do the extra spaces come from auto-formatting? : D

Reply:
I guess they are intentional. This was directly copy/pasted from the query Esa provided. I have in mind to improve the queries with a helper that would generate these fields (the low-level support exists in `stringLiteralUnionFields` function in `model.ts`).

Reply:
Ohhhh that'll be really neat!

I took a quick look at `stringLiteralUnionFields`, got dizzy before digging further :sweat_smile:  🥴